swarm/network/protocol_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	p2ptest "github.com/ethereum/go-ethereum/p2p/testing"
 )
 
+const DefaultVersion = 5
```

Comment:
I think we need a better name for this, and a godoc - what is this `DefaultVersion`.

Reply:
Changed it to TestProtocolVersion